lib/Dialect/TTNN/Utils/PassOverrides.cpp

```diff
@@ -6,6 +6,18 @@
 
 namespace mlir::tt::ttnn {
 
+const std::string OptionNames::optimizerPassEnabled = "enable-optimizer";
```

Comment:
I'm guessing that there is a problem when reading these from pipeline options?

Reply:
Yes. We can't bind class TTIRToTTNNBackendPipelineOptions, we build python lib with RTTI, and if the structure has or inherits some virtual methods, something wrong happens with the linker. We can talk more about this with Stefi and Topalko, about what we can do in the future, but I think we can't do too much. If we want to expose something from MLIR to Python in this way, we always need to have some simple(C++) structure, or to extend pybind and LLVM. There is also, llvmpy lib, but often they have the same issues with the linking.